docker/docker_image_dest.go

```diff
@@ -78,10 +78,10 @@ func (d *dockerImageDestination) SupportsSignatures(ctx context.Context) error {
 		return err
 	}
 	switch {
-	case d.c.signatureBase != nil:
-		return nil
 	case d.c.supportsSignatures:
 		return nil
+	case d.c.signatureBase != nil:
+		return nil
 	default:
```

Comment:
This is now unreachable; so this function can just be `return nil`, or the error returned might be “Internal error”+description of the inconsistency.

Reply:
I keep them as internal errors.